File: archive/find-target-indices-after-sorting-array.js
const targetIndices = function (nums, target) {
  let c = 0
  let n = 0
  nums.forEach(el => {
    if (el === target) {
      n++
    } else if (el < target) {
      c++
    }
  })
  let ans = []
  for (let i = c; i < c + n; i++) {
    ans.push(i)
  }

  return ans
};

console.log(targetIndices([1, 2, 5, 2, 3], 2))

File: go-archive/find-target-indices-after-sorting-array.go
func targetIndices(nums []int, target int) []int {
    sort.SliceStable(nums, func (i int, j int)bool {
        return nums[i] < nums[j]
    })

    ans:=[]int{}
    for i:=0; i < len(nums); i++ {
        if nums[i] == target {ans = append(ans, i)}
    }

    return ans
}